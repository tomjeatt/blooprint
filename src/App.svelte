<script lang="ts">
    import Card from './components/Card/Card.svelte';
    import store from './stores/grid';
    const createCard = () => {
        store.addCard({
            id: `${Math.random()}`,
            text: 'New Card Text',
        });
    };
</script>

<main>
    <h1>Blooprint</h1>
    <p>A service blueprinting application.</p>
    {#if $store.cards.length}
        {#each $store.cards as card}
            <Card />
        {/each}
    {/if}

    <button on:click={createCard}> + </button>
</main>
