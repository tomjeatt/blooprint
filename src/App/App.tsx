import React from 'react';
import Theme from '../theme/Theme';

const App: React.FC = () => {
  return (
    <main>
        <Theme>
          <h1>Blooprint</h1>
        </Theme>
    </main>
  );
};

export default App;
