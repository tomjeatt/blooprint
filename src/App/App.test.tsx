import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from '.';

afterEach(cleanup);

describe('Application renders', () => {
  it('renders', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
