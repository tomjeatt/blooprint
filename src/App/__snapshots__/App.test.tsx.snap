// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application renders renders 1`] = `
<DocumentFragment>
  <main>
    <h1>
      Blooprint
    </h1>
  </main>
</DocumentFragment>
`;
